Components/FeatureGoogleAnalytics/README.md

```diff
@@ -1,3 +1,10 @@
-# Feature Google Analytics
+# Google Analytics
 
-If a Google Analytics ID is provided in Global Options, the tracking code will be added to all pages, expect for logged in WordPress admins. Entering "debug" instead will log to the browser console only.
+Enable the Google Analytics tracking configuration in the WordPress back end.
+
+If a valid Google Analytics ID is provided, the tracking code will automatically be added to all pages.
+
+The following options are provided under the **"Global Options"** menu:
+
+- Google Analytics ID: This must be a valid Google Analytics ID.
+- Anonymize IP: Check to enable the Google Analytics `_anonymizeIp` function and anonymize the last digits of the user's IP.
```

Comment:
Please keep the shorter readme (I just updated it 2 days ago). But could you please fix my typo: "expect -> except"?

Reply:
I added the readme from the previous version and fixed the typo.